crates/generate/src/funcs.rs

```diff
@@ -313,7 +231,6 @@ fn marshal_arg(
             let handle_type = names.type_ref(tref, anon_lifetime());
             quote!( let #name = #handle_type::from(#name); )
         }
-        _ => unimplemented!("argument type marshalling"),
```

Comment:
Are we sure to remove this at this stage? Wouldn't it be a useful reminder when the spec get extended?

Reply:
Oh this can be removed because the match is exhaustive now, and anything new being added will be caught at compile time rather than runtime.